media-sound/mac/mac-4.38.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A fast and powerful lossless audio compressor"
+DESCRIPTION="Monkey's Audio is a lossless audio format. This package provides \
+the mac command line utility for compressing and decompressing Monkey's Audio \
+files."
+HOMEPAGE="https://www.monkeysaudio.com/"
+COPYRIGHT="2000-2018 Matthew T. Ashland"
+LICENSE="Monkey's Audio SDK and Source Code License Agreement"
+REVISION="1"
+SOURCE_URI="https://www.monkeysaudio.com/files/MAC_SDK_438.zip"
+CHECKSUM_SHA256="1ab9a0c74a0478c3fe713908789017c53c3b333a482225d2d09990a16069f981"
+SOURCE_DIR=""
+PATCHES="mac-$portVersion.patchset"
+ADDITIONAL_FILES="Makefile"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="${portVersion}"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	mac$secondaryArchSuffix = $portVersion
+	cmd:mac = $portVersion
+	lib:libMAC$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	mac${secondaryArchSuffix}_devel = $portVersion
+	devel:libMAC$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	mac$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	cp "$portDir"/additional-files/Makefile Makefile
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make prefix="$prefix" bindir="$commandBinDir" libdir="$libDir" includedir="$includeDir" install
+
+	prepareInstalledDevelLibs libMAC
```

Comment:
Depending on what was chosen (between A and B) previously:

A)
```diff
- 	prepareInstalledDevelLibs libMAC
+ 	prepareInstalledDevelLib libMAC
```

B)
```diff
-	prepareInstalledDevelLibs libMAC
+	prepareInstalledDevelLib libMAC
+
+	if [ -n "$commandSuffix" ]; then
+		rm -rf "$commandBinDir"
+	fi
```

You can keep the "s" in prepareInstalledDevel**Libs** if you prefer.

Reply:
Opted for variant A and updated.